src/components/CurriculumComponents/CurriculumDownloadTab/CurriculumDownloadTab.test.tsx

```diff
@@ -68,12 +68,10 @@ describe("Component - Curriculum Download Tab", () => {
     test("user can see the tier selector for secondary maths", async () => {
       // NOTE: This is only active during testing.
       mockPrerelease("curriculum.downloads");
-      const { findByText, findByTestId } = renderComponent({
+      const { findByTestId } = renderComponent({
         tiers: tiersMock,
       });
-      const formHeading = await findByText("Download");
       const tierSelector = await findByTestId("tier-selector");
-      expect(formHeading).toBeInTheDocument();
       expect(tierSelector).toBeInTheDocument();
```

Comment:
@assadk88 i've released a new version of oak-components with a minor change. Updating past your release in https://github.com/oaknational/oak-components/pull/298 i've got this broken test. It seems that the heading has been removed. Am I okay to release or do you need to release https://github.com/oaknational/Oak-Web-Application/pull/2852 to avoid a breaking change? It's a patch release so am I ok?